ab-testing/scripts/validation/validExpiration.test.ts

```diff
@@ -0,0 +1,52 @@
+import { assertEquals } from 'jsr:@std/assert/equals';
+import { allExpirationsValid } from './validExpiration.ts';
+import { ABTest } from '../../types.ts';
+import { assertThrows } from 'jsr:@std/assert/throws';
+
+function futureDay() {
+	const today = new Date();
+	today.setDate(today.getDate() + 1);
+	return today.toISOString().split('T')[0] as ABTest['expirationDate']; // Format as YYYY-MM-DD
+}
+
+function pastDay() {
+	const pastDate = new Date();
+	pastDate.setDate(pastDate.getDate() - 2);
```

Comment:
why -2 and not -1?

Reply:
why not :p

Reply:
Sorry. Just noticed it was a test not the actual function itself 😓